Tropos/Controllers/TRApplicationController.m

```diff
@@ -40,4 +47,12 @@ - (void)setMinimumBackgroundFetchIntervalForApplication:(UIApplication *)applica
     }
 }
 
+- (TRCourierClient *)courier;
+{
+    if (!_courier) {
+        _courier = [[TRCourierClient alloc] initWithApiToken: TRCourierAPIToken];
```

Comment:
Was there a specific reason this isn't in `-init`? I think it would be a good idea to follow the existing initialization pattern where appropriate.